### DISCUSSION TITLE
Target Rate Sensor Automation Question

### DISCUSSION THREAD

**Discussion by dodavies** [BottlecapDave/HomeAssistant-OctopusEnergy#372] [Q&A] [answered]
I am on agile now as my go tariff just ended so I'm trying to setup charging two EV's based on the lowest of the low rates in agile

My target sensor is setup as intermittent for 6 hours from 23:00

Its set to six hours as we have three phase and the Tesla never normally needs more than that but that could be removed I guess.
The sensor goes on/off but I can't figure out how to only charge if the rate is under five pence.

I have an input boolean called PowerCheap which is toggled by the target sensor going on/off

I have two automations one that starts and another that stops charging

<img width="1045" alt="Screenshot 2023-08-13 at 16 32 56" src="https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/10104862/9448e986-dfbd-4353-9050-b94d21c19cd1">
<img width="1114" alt="Screenshot 2023-08-13 at 16 37 44" src="https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/10104862/18aeff6c-4a7d-442e-8782-5594727b89b4">

I thought about adding a condition to the start automation to check if the rate is under 0.05 but can't seem to get it figured out

Using developer tools to check sensor.octopus_energy_electricity_99e9999999_9999999999999_current_rate 
As I write it is 0.263445 but I can't seem to set a condition that accepts 0.05
Is there a way anyone can think of to do this? or is there something I can do in this integration to make a sensor that only goes on if its less than x pence ?

**Comment by BottlecapDave** [comment, marked as answer]
Hi there. If you're wanting to not turn on above a certain value, you should be able to just add a numeric state condition under "conditions" to your automation looking at the current rate sensor with an below value of `0.06` (as I believe the value is not inclusive). It looks like there's a bug in the UI that's preventing decimal figures, so you need to enter it in YAML mode `accessed via the three dots in the condition).

**Comment by dodavies** [comment]
Thanks David, yes it lets me add it in yaml mode.
